common/src/test/java/org/apache/sedona/common/ConstructorsTest.java

```diff
@@ -37,6 +37,23 @@ public void geomFromWKT() throws ParseException {
         ParseException invalid = assertThrows(ParseException.class, () -> Constructors.geomFromWKT("not valid", 0));
         assertEquals("Unknown geometry type: NOT (line 1)", invalid.getMessage());
     }
+
+    @Test
+    public void geomFromEWKT() throws ParseException {
+        assertNull(Constructors.geomFromEWKT(null));
+
+        Geometry geom = Constructors.geomFromEWKT("POINT (1 1)");
```

Comment:
Please add one more test case `SRID=4269;POINT (1 1)` (no space between semicolon and the wkt string)